smoke-test/src/test/java/org/opennms/smoketest/BSMAdminIT.java

```diff
@@ -28,70 +28,49 @@
+    private final String BASIC_SERVICE_NAME = "BasicService";
+    private final String BASIC_SERVICE_NAME_TO_CHANGE = "BasicServiceToChange";
 
-        // Create a new Business Service via the REST API
-        String servicePrefix = UUID.randomUUID().toString();
-        createServiceWithPrefix(servicePrefix);
-
-        // Verify that the Business Service appears in the list
-        m_driver.get(BASE_URL + "opennms/admin/bsm/index.jsp");
-        findElementByXpath("//*[contains(text(), '" + servicePrefix + "-name')]");
-
-        // The Business Service's attributes should also be displayed
-        WebElement el = findElementByXpath("//*[contains(text(), '" + servicePrefix + "-key')]");
-        WebElement parent = el.findElement(By.xpath(".."));
-        assertEquals(servicePrefix + "-key: " + servicePrefix + "-value", parent.getText());
+    @Test
+    public void canShowNoBusinessServicesPresentText() {
+        m_driver.get(BSM_LIST_URL);
+        findElementByXpath("//*[contains(text(), 'There are no business services defined.')]");
     }
 
-    /**
-     * Creates a BusinessService with a single attribute.
-     *
-     * The name of the service, and its key-value attributes are all prefixed
-     * with the given value.
-     */
-    private static void createServiceWithPrefix(String prefix) throws ClientProtocolException, IOException {
-        String businessServiceApiUrl = BASE_URL + "opennms/api/v2/business-services";
-
-        try (HttpClientWrapper httpClient = HttpClientWrapper.create()) {
-            httpClient.addBasicCredentials(BASIC_AUTH_USERNAME, BASIC_AUTH_PASSWORD);
-            httpClient.usePreemptiveAuth();
-            HttpPost request = new HttpPost(businessServiceApiUrl);
-            request.setHeader(HttpHeaders.CONTENT_TYPE, MediaType.JSON_UTF_8.toString());
+    @Test
+    public void canCreateBusinessServices() {
+        m_driver.get(BSM_LIST_URL);
+        findElementById("createInput").sendKeys(BASIC_SERVICE_NAME);
+        findElementById("createButton").click();
+        findElementByXpath("//*[contains(text(), '" + BASIC_SERVICE_NAME + "')]");
+    }
 
-            StringEntity params = new StringEntity("{" +
-                    "\"name\":\"" + prefix + "-name\"," +
-                    "\"attributes\":{\"attribute\":[{\"key\":\"" + prefix + "-key\",\"value\":\"" + prefix + "-value\"}]}" +
-                    "}");
-            request.setEntity(params);
+    @Test
+    public void canDeleteBusinessServices() {
+        m_driver.get(BSM_LIST_URL);
+        findElementById("delete-" + BASIC_SERVICE_NAME).click();
+        findElementByXpath("//*[contains(text(), 'There are no business services defined.')]");
+    }
 
-            try (CloseableHttpResponse response = httpClient.execute(request)) {
-                assertEquals(201, response.getStatusLine().getStatusCode());
-            }
-        }
+    @Test
+    public void canEditBusinessServices() {
```

Comment:
Can you please seperate the code segments and add a comment to each of the segments what it is doing?
E.g.:

    //create new service
    ...
    // rename created service
    ...
    // delete created service
    ...